content/SCALE/SCALEUIReference/Storage/Datasets/DatasetsScreensScale.md

```diff
@@ -91,6 +89,8 @@ A child dataset path displays the root dataset (pool) name and parent dataset.
 
 **Edit** opens the **[Edit Dataset](#add-and-edit-dataset-screens)** screen for the selected dataset.
 
+The **Promote** button appears on the **Dataset Details** widget if you clone a dataset from an existing snapshot. The dataset cloned from the snapshot also displays in the **Datasets** listing. Promoting the cloned child dataset allows users to delete the parent volume that created the clone. Otherwise, you cannot delete a clone while the original volume still exists. See [zfs-promote.8](https://openzfs.github.io/openzfs-docs/man/8/zfs-promote.8.html).
```

Comment:
for a parallel construction in how the other options (buttons) are described....
```suggestion
**Promote** appears on the **Dataset Details** widget when you select a cloned snapshot on the dataset tree table. This option promotes the cloned child dataset and allows users to delete the parent volume that created the clone. Otherwise, you cannot delete a clone while the original volume still exists. See [zfs-promote.8](https://openzfs.github.io/openzfs-docs/man/8/zfs-promote.8.html).
```